grafana-plugin/e2e-tests/globalSetup.ts

```diff
@@ -21,6 +28,13 @@ import { goToGrafanaPage } from './utils/navigation';
 
 const grafanaApiClient = new GrafanaAPIClient(GRAFANA_ADMIN_USERNAME, GRAFANA_ADMIN_PASSWORD);
 
+enum OrgRole {
+  None = 'None',
+  Viewer = 'Viewer',
+  Editor = 'Editor',
+  Admin = 'Admin',
+}
```

Comment:
what is this change related to?

Reply:
We don't mark the package.json to be a module, while grafana I believe it does, which leads to conflicts when importing it. Playwright will complain on the import from `grafana/data` of `OrgRole`, you'll get this error

`Error [ERR_REQUIRE_ESM]: require() of ES Module not supported [duplicate]`

Which would sort of be fixed by adding this to the `package.json`

`"type": "module"`

but this means we're only allow to use ES6 imports, therefore no `require` imports anymore, and I believe that would be conflicting with the libraries we depend on that use `requrie` in their code.

I'm not an expert on this aspects, so this might need additional research, for now I've removed the import and just set the enum in our codebase instead, couldn't figure out a better way instead :thinking: Changing the codebase to `module` leads to other problems